src/main/java/com/faforever/client/fa/LaunchCommandBuilder.java

```diff
@@ -303,7 +304,9 @@ public LaunchCommandBuilder gameOptions(Map<String, String> gameOptions) {
   }
 
   public LaunchCommandBuilder executableDecorator(String executableDecorator) {
-    this.executableDecorator = Strings.isNullOrEmpty(executableDecorator) ? DEFAULT_EXECUTABLE_DECORATOR : executableDecorator;
+    boolean linuxProton = Arrays.asList(FafClientApplication.getAdditionalProfiles()).contains(FafClientApplication.PROFILE_LINUX);
+    String profileLauncher = (linuxProton ? "protontricks-launch --appid 9420 " : "") + DEFAULT_EXECUTABLE_DECORATOR;
+    this.executableDecorator = Strings.isNullOrEmpty(executableDecorator) ? profileLauncher : executableDecorator;
     return this;
   }
```

Comment:
This should not really be in the launch command builder it should be moved up to where the builder is constructed.

Reply:
I can learn from example, but I cannot accomplish this upstream without making a bigger mess without knowing more.

Reply:
isn't defaultLaunchCommand that choke point?